config/locales/javascript/javascript.en.yml

```diff
@@ -246,6 +246,7 @@ en:
     notifications:
       mark_read: "Mark read"
       mark_unread: "Mark unread"
+      new_notifications: "You have new notifications"
```

Comment:
Thanks for pointing this out. Actually we should be using `count` with `one` -> "... notification" `other` -> "... notifications".